cmd/root.go

```diff
@@ -151,7 +151,9 @@ var rootCmd = &cobra.Command{
 			Scheme:          mgr.GetScheme(),
 			ControllerClass: controllerClass,
 			RequeueInterval: storeRequeueInterval,
-		}).SetupWithManager(mgr); err != nil {
+		}).SetupWithManager(mgr, controller.Options{
+			MaxConcurrentReconciles: concurrent,
+		}); err != nil {
```

Comment:
This speeds up e2e tests and allows for faster reconciliation of secret stores (that's a problem in larger setups when the controller restarts).

I'd like to increase the `--concurrent` default from 1 to 10. I think this would improve the "out-of-the-box" performance